menas/src/main/scala/za/co/absa/enceladus/menas/WebSecurityConfig.scala

```diff
@@ -78,31 +82,38 @@ class WebSecurityConfig {
         .csrf()
           .ignoringAntMatchers("/api/login")
         .and()
-          .exceptionHandling()
-          .authenticationEntryPoint(restAuthenticationEntryPoint)
+        .exceptionHandling()
+                .authenticationEntryPoint(spnegoEntryPoint())
         .and()
-          .authorizeRequests()
-          .antMatchers("/index.html", "/resources/**", "/generic/**",
-            "/service/**", "/webjars/**", "/css/**", "/components/**")
-          .permitAll()
+        .authorizeRequests()
+        .antMatchers("/index.html", "/resources/**", "/generic/**",
+          "/service/**", "/webjars/**", "/css/**", "/components/**")
+        .permitAll()
```

Comment:
I have a feeling someone tabbed this the other way in a PR just a week ago.

Reply:
I'm gonna tab it in such a way to highlight the different config sections... autoformat screws it up